cpmonitor/models.py

```diff
@@ -314,7 +325,12 @@ def validate_constraints(self, exclude=None):
         This is necessary, because `editable=False` excludes the field from validation
         and UniqueConstraint does not allow error messages showing content.
         """
-        exclude.remove("slugs")
+
+        if exclude and "slugs" in exclude:
+            if "slugs" in exclude:
+                exclude.remove("slugs")
+            if "city" in exclude:
+                exclude.remove("city")
```

Comment:
The `"slugs" in exclude` condition is duplicated. One of them can be removed. Which one depends on if you intended "city" to be removed only if "slugs" was excluded, but I can't see why..?
```suggestion
        if exclude:
            if "slugs" in exclude:
                exclude.remove("slugs")
            if "city" in exclude:
                exclude.remove("city")
```
or
```suggestion
        if exclude and "slugs" in exclude:
            exclude.remove("slugs")
            if "city" in exclude:
                exclude.remove("city")
```